scalding-core/src/main/scala/com/twitter/scalding/serialization/Boxed.scala

```diff
@@ -50,7 +358,103 @@ object Boxed {
+    ({ t: Any => new Boxed51(t) }, classOf[Boxed51[Any]]),
+    ({ t: Any => new Boxed52(t) }, classOf[Boxed52[Any]]),
+    ({ t: Any => new Boxed53(t) }, classOf[Boxed53[Any]]),
+    ({ t: Any => new Boxed54(t) }, classOf[Boxed54[Any]]),
+    ({ t: Any => new Boxed55(t) }, classOf[Boxed55[Any]]),
+    ({ t: Any => new Boxed56(t) }, classOf[Boxed56[Any]]),
+    ({ t: Any => new Boxed57(t) }, classOf[Boxed57[Any]]),
+    ({ t: Any => new Boxed58(t) }, classOf[Boxed58[Any]]),
+    ({ t: Any => new Boxed59(t) }, classOf[Boxed59[Any]]),
+    ({ t: Any => new Boxed60(t) }, classOf[Boxed60[Any]]),
+    ({ t: Any => new Boxed61(t) }, classOf[Boxed61[Any]]),
+    ({ t: Any => new Boxed62(t) }, classOf[Boxed62[Any]]),
+    ({ t: Any => new Boxed63(t) }, classOf[Boxed63[Any]]),
+    ({ t: Any => new Boxed64(t) }, classOf[Boxed64[Any]]),
+    ({ t: Any => new Boxed65(t) }, classOf[Boxed65[Any]]),
+    ({ t: Any => new Boxed66(t) }, classOf[Boxed66[Any]]),
+    ({ t: Any => new Boxed67(t) }, classOf[Boxed67[Any]]),
+    ({ t: Any => new Boxed68(t) }, classOf[Boxed68[Any]]),
+    ({ t: Any => new Boxed69(t) }, classOf[Boxed69[Any]]),
+    ({ t: Any => new Boxed70(t) }, classOf[Boxed70[Any]]),
+    ({ t: Any => new Boxed71(t) }, classOf[Boxed71[Any]]),
+    ({ t: Any => new Boxed72(t) }, classOf[Boxed72[Any]]),
+    ({ t: Any => new Boxed73(t) }, classOf[Boxed73[Any]]),
+    ({ t: Any => new Boxed74(t) }, classOf[Boxed74[Any]]),
+    ({ t: Any => new Boxed75(t) }, classOf[Boxed75[Any]]),
+    ({ t: Any => new Boxed76(t) }, classOf[Boxed76[Any]]),
+    ({ t: Any => new Boxed77(t) }, classOf[Boxed77[Any]]),
+    ({ t: Any => new Boxed78(t) }, classOf[Boxed78[Any]]),
+    ({ t: Any => new Boxed79(t) }, classOf[Boxed79[Any]]),
+    ({ t: Any => new Boxed80(t) }, classOf[Boxed80[Any]]),
+    ({ t: Any => new Boxed81(t) }, classOf[Boxed81[Any]]),
+    ({ t: Any => new Boxed82(t) }, classOf[Boxed82[Any]]),
+    ({ t: Any => new Boxed83(t) }, classOf[Boxed83[Any]]),
+    ({ t: Any => new Boxed84(t) }, classOf[Boxed84[Any]]),
+    ({ t: Any => new Boxed85(t) }, classOf[Boxed85[Any]]),
+    ({ t: Any => new Boxed86(t) }, classOf[Boxed86[Any]]),
+    ({ t: Any => new Boxed87(t) }, classOf[Boxed87[Any]]),
+    ({ t: Any => new Boxed88(t) }, classOf[Boxed88[Any]]),
+    ({ t: Any => new Boxed89(t) }, classOf[Boxed89[Any]]),
+    ({ t: Any => new Boxed90(t) }, classOf[Boxed90[Any]]),
+    ({ t: Any => new Boxed91(t) }, classOf[Boxed91[Any]]),
+    ({ t: Any => new Boxed92(t) }, classOf[Boxed92[Any]]),
+    ({ t: Any => new Boxed93(t) }, classOf[Boxed93[Any]]),
+    ({ t: Any => new Boxed94(t) }, classOf[Boxed94[Any]]),
+    ({ t: Any => new Boxed95(t) }, classOf[Boxed95[Any]]),
+    ({ t: Any => new Boxed96(t) }, classOf[Boxed96[Any]]),
+    ({ t: Any => new Boxed97(t) }, classOf[Boxed97[Any]]),
+    ({ t: Any => new Boxed98(t) }, classOf[Boxed98[Any]]),
+    ({ t: Any => new Boxed99(t) }, classOf[Boxed99[Any]]))
 
```

Comment:
can you change the type below on next:

`def next[K]: (K => Boxed[K], Class[_ <: Boxed[K]])`

That is actually more correct since Class is invariant.